packages/core/src/services/evaluations/prompt/index.test.ts

```diff
@@ -98,4 +98,26 @@ describe('getEvaluationPrompt', () => {
     expect(metadata.errors.length).toBe(0)
     expect(metadata.config.model).toBe(model)
   })
+
+  it('returns evaluation description for manual evaluations', async () => {
```

Comment:
Getting a prompt for a manual eval does not fail. Just return evaluation description.